praktikum/poglavlja/09_strukture/index.rst

```diff
@@ -0,0 +1,161 @@
+Strukture
+=========
+
+Podeli svoje utiske o poglavlju na anketi `OVDE <https://docs.google.com/forms/d/1hmqQGSA6yqIBEBoK_4nKHm3qV2cvQsszgNvieOi9APU>`_.
+
+Preduslovi za rad:
+
+.. literalinclude:: objasnjenja/preduslovi-za-rad.txt
+   :language: none
+
+Predstavljaju složene tipove podataka koji se prave kombinovanjem više primitivnih tipova i/ili ranije formiranih složenih tipova.
+Struktura je skup istih ili različitih tipova podataka koji se čuvaju pod istim imenom. Koriste se kada imamo vrednosti
```

Comment:
Mislim da ova rečenica može da se izbaci.

Reply:
Prva?

Reply:
mozda popravljeno, proveri

Reply:
"Struktura je skup istih ili različitih tipova podataka koji se čuvaju pod istim imenom.", ova rečenica je rečenica pre nje ponovljena na drugačiji način. Nema nikakvu dodatnu vrednost i zato sam mislio da je izbaciš i zbog ovog "istih ili različitih tipova podataka", to mi nije baš najjasnije šta si želela da kažeš.